middlewares/authentication.js

```diff
@@ -19,6 +23,13 @@ const verifyToken = (req, res, next) => {
       return next(error);
     }
 
+    const user = await User.findOne({ where: { id: decoded.id } });
```

Comment:
Could you use the `findByPk` method here instead? I have noticed that we hardly have used it, but the good thing is since `id` is the primary key, we can do without the `where` clause. Hence, a simple `User.findByPk(decoded.id)` should suffice.

Reply:
Thanks, I've made the change